### ISSUE TITLE
[OpenBSD][cpu] errors and incorrect results

### ISSUE THREAD

**Issue by omar-polo** [shirou/gopsutil#646]
**Describe the bug**
There are a couple of things that don't work on OpenBSD:

- [`cpu.Info`](https://github.com/shirou/gopsutil/blob/master/cpu/cpu_openbsd.go#L137) fails because `unix.SysctlUint32("hw.ncpuonline")` fails
   This happens both on OpenBSD 6.3 (i386) and OpenBSD-CURRENT (amd64). atm I cannot test on other versions/archs. I'm not too familiar with the sysctl api, but I'll try to look into that.

- `cpu.Percent`: if `percpu` is true, the library returns n times the percentages of the average of all cpus, not the individual cpu percentage ([see here](https://github.com/shirou/gopsutil/blob/master/cpu/cpu_openbsd.go#L87)). I'll make a pr for that. However, even after the fix, the results are strange: the cpu1 and cpu3 are always at 0%. Again, I'm not familiar with sysctl, so while I'll try to fix that, help or ideas are welcome =)

**To Reproduce**
```go
// perc.go
package main

import (
        "fmt"
        "log"
        "time"

        "github.com/shirou/gopsutil/cpu"
)

func main() {
        percents, err := cpu.Percent(2*time.Second, true)
        if err != nil {
                log.Fatal(err)
        }

        fmt.Println(percents)
}
```

```sh
for i in `jot 10`; do sleep 1; go run perc.go; done # jot is like linux' seq
# sample output:
[10.555555556157394 0 8.55614973282839 0]
[14.285714285879273 0 10.052910052573967 0]
[14.054054053898863 0 13.297872340392594 0]
[26.94610778451984 0 19.879518072701064 0]
[10.58201058254832 0 10.52631578989694 0]
[14.361702127453713 0 7.3298429320049845 0]
[10.582010581778377 0 10.994764398388417 0]
[28.97727272778009 0 27.071823204744145 0]
[3.7433155084042373 0 10.309278350043751 0]
[9.8958333335386 0 3.6269430051793936 0]
```

**Expected behavior**
- `cpu.Info` should return without erros
- `cpu.Percent` should return percentages consistent with what `top` reports

**Environment:**
 - [x] OpenBSD: `uname -a`: `OpenBSD venera.rosetta.moe 6.5 GENERIC.MP#760 amd64`
Note that while the version reported is 6.5, this is a ~7 days old snapshot

**Comment by omar-polo** [comment]
Sorry, I noticed now that I haven't been clear: the output sample is *after* merging #647.

**Comment by Lomanic** [comment]
According to https://github.com/shirou/gopsutil/pull/647#issuecomment-474538518, this issue is now fixed with #647 being merged, right @omar-polo?

**Comment by omar-polo** [comment]
Yup, I forgot to close, sorry.
